waku/v2/waku_archive/driver/postgres_driver/postgres_driver.nim

```diff
@@ -0,0 +1,288 @@
+when (NimMajor, NimMinor) < (1, 4):
+  {.push raises: [Defect].}
+else:
+  {.push raises: [].}
+
+import
+  std/db_postgres,
+  std/strformat,
+  std/nre,
+  std/options,
+  std/strutils,
+  stew/[results,byteutils],
+  chronos
+
+import
+  ../../../waku_core,
+  ../../common,
+  ../../driver
+
+export postgres_driver
+
+type PostgresDriver* = ref object of ArchiveDriver
+  connection: DbConn
+  preparedInsert: SqlPrepared
+
+proc dropTableQuery(): string =
+  "DROP TABLE messages"
+
+proc createTableQuery(): string =
```

Comment:
Curious if we can manage migrations in plain SQL, so that other implementations can use the same migrations (not a blocker, just want to know your thoughts)

Reply:
Good point @rymnc ! I'll set that as a pending action item in the issue